docs/conf.py

```diff
@@ -42,7 +42,8 @@
     "sphinx.ext.intersphinx",
     "sphinx.ext.todo",
     "notfound.extension",
-    "sphinxcontrib.towncrier",
+    # Disabled until it speaks Markdown:
```

Comment:
@hynek I'm pretty sure it should still work. What problem are you experiencing?

Reply:
It included the Markdown rendered as rST including complains about syntax in the console. 

I tried setting a parser like with include but that didn't work. 

I suspect this might not be too hard to fix?

Reply:
I've seen reports of MyST users using the extension + tried it out myself at some point, and it worked. It only invokes `towncrier --draft` and then feeds the output to the docutils/Sphinx source parser, without specifying any markup — my understanding is that it can be parsed with any parsers that are configured in Sphinx.

I'll try to look into your PR deeper to see what's happening.

Reply:
So... I haven't checked out this branch locally, but I'm pretty sure it's https://github.com/python-attrs/attrs/pull/1067#pullrequestreview-1202439985.